tests/HUnitTestExtended.hs

```diff
@@ -1,3 +1,6 @@
+{-# OPTIONS_GHC -Wno-unrecognised-warning-flags #-}  -- since GHC 8.0
+{-# OPTIONS_GHC -Wno-x-partial #-}                   -- since GHC 9.8
```

Comment:
Is `x-partial` included in `-Wdefault`?  If not, then maybe use `-w  -Wdefault`.

Reply:
Something I just noticed, all this tests are technically broken.

    `seq` return ()

doesn't really work. This should be `evaluate` instead.  If you wanna fix this too, then that would be more than welcome.